modules/agent-install-networking.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent/installing-with-agent.adoc
+
+:_content-type: CONCEPT
+[id="agent-install-networking_{context}"]
+= Networking
+
+During the initial boot, the machines require an IP address configuration that is set either through a Dynamic Host Configuration Protocol (DHCP) server or statically by providing the required boot options. After a network connection is established, the machines download their Ignition config files from an HTTP or HTTPS server.
+
+.DHCP
+
+* You must specify only the `rendezvousIP` parameter when you use the preferred method of `install-config.yaml` and `agent.config.yaml` files:
+
++
+.Sample agent-config.yaml.file
+
+[source,yaml]
+----
+  cat > agent-config.yaml << EOF
+  kind: AgentConfig
+  metadata:
+    name: sno-cluster
+  rendezvousIP: 192.168.111.80 <1>
+----
+<1> Required for DHCP.
+
+.Static networking
+
+* You must specify the `rendezvousIP` and `networkConfig` parameters when you use the preferred method of `install-config.yaml` and `agent.config.yaml` files:
```

Comment:
For static networking, `rendezvousIP` is not needed but `networkConfig` is because the agent installer internally generates a low level yaml file i.e. nmstateconfig.yaml based on the `networkConfig` data from agent-config.yaml in this case.

Reply:
I have removed `rendezvousIP`.

Reply:
I have re-structured the content based on your first comment